src/pages/clients/create/Create.tsx

```diff
@@ -94,6 +97,16 @@ export default function Create() {
       .then((response) => {
         toast.success('created_client');
 
+        queryClient.invalidateQueries('/api/v1/clients');
+
+        window.dispatchEvent(
+          new CustomEvent('invalidate.combobox.queries', {
+            detail: {
+              url: endpoint('/api/v1/clients'),
+            },
+          })
+        );
```

Comment:
I don't think this is a thing anymore?

Reply:
Yeah u are right, I missed that we do not need this anymore. Removed!